superset-frontend/src/views/CRUD/welcome/ChartTable.tsx

```diff
@@ -155,7 +155,10 @@ function ChartTable({
             name: 'View All »',
             buttonStyle: 'link',
             onClick: () => {
-              window.location.href = '/chart/list';
+              window.location.href =
+                chartFilter === 'Favorite'
+                  ? '/chart/list/?filters=(favorite:!t)'
+                  : '/chart/list/';
```

Comment:
Can we use react-router for this so we don't have to wait for a full page reload?